CHANGELOG.rst

```diff
@@ -28,6 +28,9 @@ Added
 * Add ``ST2_USE_DEBUGGER`` env var as alternative to the ``--use-debugger`` cli flag. #5675
   Contributed by @cognifloyd
 
+* Added purging of old tokens. #56791
+  Contributed by Amanda McGuinness (@amanda11 intive)
```

Comment:
Are we using the `(@username <orgname>)` anywhere else in the changelogs?

Reply:
I think there are a few examples. It's really up to the person who wrote the PR. Some people opt not to include a "Contributed by" bit in the changelog. Others need to credit their employer (intive in the case of Amanda).